patches/net/minecraft/world/item/alchemy/PotionContents.java.patch

```diff
@@ -0,0 +1,13 @@
+--- a/net/minecraft/world/item/alchemy/PotionContents.java
++++ b/net/minecraft/world/item/alchemy/PotionContents.java
+@@ -169,6 +_,10 @@
+             p_331296_.accept(NO_EFFECT);
+         }
+ 
++        // Neo: Override handling of potion attribute tooltips to support IAttributeExtension
```

Comment:
Instead of putting this here, do we want to instead do it after vanilla's `whenDrank` line adding so that we don't have to maintain a copy of vanilla in doing so?

Reply:
Sure, that matches some recent logic changes to `applyTextFor` where the code adding the blank line and group name (which is `whenDrank` in this case) to the method one level up.